src/FooterToolbar/index.tsx

```diff
@@ -1,15 +1,83 @@
-import { VNodeChild } from 'vue';
+import {
+  computed,
+  defineComponent,
+  inject,
+  onBeforeUnmount,
+  onMounted,
+  PropType,
+  VNodeChild,
+} from 'vue';
+import { defaultProProviderProps, injectProConfigKey } from '../ProProvider';
+import { RouteContextProps, useRouteContext } from '../RouteContext';
 import './index.less';
 
-export interface FooterToolbarProps {
-  extra?: VNodeChild;
+export interface IFooterToolbarProps {
+  extra?: VNodeChild | JSX.Element;
   renderContent?: (
-    props: FooterToolbarProps & { leftWidth?: string },
+    props: this & RouteContextProps & { leftWidth?: string },
```

Comment:
this ?

Reply:
Same as FooterToolbarProps, but I' ll changed back.